src/game/client/components/menus.cpp

```diff
@@ -9,12 +9,12 @@
 #include <engine/config.h>
 #include <engine/editor.h>
 #include <engine/engine.h>
-#include <engine/friends.h>
 #include <engine/keys.h>
 #include <engine/serverbrowser.h>
 #include <engine/storage.h>
 #include <engine/textrender.h>
 #include <engine/shared/config.h>
+#include <engine/client/friends.h>
```

Comment:
Hm, could you move the enums to the interface? It shouldn't include sth from `engine/client/` but work with the interfaces instead.

Reply:
Yes, that makes sense